components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt

```diff
@@ -41,6 +41,14 @@ class GleanDebugActivity : Activity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (!Glean.isInitialized()) {
```

Comment:
Does this work even if the application is started by the intent?

Reply:
It seems to.  i.e. I tested by killing the app in the application manager and then running the `adb` command (on a version of `sample-glean` with `Glean.initialize` commented out).

Reply:
I wonder if this is racing with the init from the main activity :\

Reply:
Well, looking into this sent me down the rabbit hole where I found #2863 which is much higher priority.  Let's get that merged then come back to this.

Reply:
This seems to work now that #2863 is implemented, so ready for another review.